### ISSUE TITLE
Village False positive with 1.7.10 profile 

### ISSUE THREAD

**Issue by BigAlanM** [toolbox4minecraft/amidst#224]
I found an interesting seed ("604679399374955521") using Amidst v4.2-beta2 and my profile for PCMCv1.10.2 (even though it doesn't report all new village locations) and was in the process of posting the discovery to the MCF Seeds forum and decided to see if the island village existed in previous MC versions.
I changed Amidst to my PCMCv1.7.10 profile and entered the seed. Amidst shows the village on the island so I created a world in PCMCv1.7.10 to verify this and the island is empty albeit exactly the same as the PCMCv1.10.2 island in all other ways.

I don't recall Amidst v4.2-beta2 giving false positives before.

EDIT: I just got the same results with my PCMCv1.8.9 profile in both Amidst and PCMC.  Amidst shows a nonexistent village.   Also the same results with the profile for PCMCv1.9.4 (which Amidst doesn't recognize).

**Comment by stefandollase** [comment]
I find it hard to follow your description. Maybe you can clarify it? In which location with which seed did you see a village in Amidst, but not in the generated Minecraft world? Which version of Amidst did you use with which version of Minecraft? Which version of Minecraft did you use?

Anyhow, [the FAQ](https://github.com/toolbox4minecraft/amidst/wiki/FAQ#why-are-the-structures-displayed-by-amidst-incorrect) states that there is a chance of 10% to 20% for Amidst to display a false positive. This information was originally taken from [the old Wiki](https://github.com/skiphs/AMIDST/wiki) and if I understand the wiki history correctly, you contributed this information by yourself, so I am a little bit confused about this issue :-P Am I getting something wrong?

**Comment by BigAlanM** [comment]
The seed "604679399374955521" when entered into Amidst using either a 1.7.10 or 1.8.9 profile produces a false positive for a village on the spawn island. 
As it turns out, the village IS created when using PCMCv1.10.2 Minecraft so this is a case where even Amidstv4.2-beta2 is correct for a 1.10.2 village placed where before MC considered the location inappropriate.

You're correct that I used to warn people that Amidst is about 80 to 90% accurate but that was quite some time ago and I had forgotten about it.  Also, I had become spoiled to 4.2-b2 being quite accurate and me not finding (or hearing about) false negatives.

**Comment by stefandollase** [comment]
I am not sure about the changes that happened since Amidst 3.7, I would have to look them up to be sure, but here is what I remember:

I did not actively improve this algorithm by myself. However, it is possible that AmidstExporter contained improvements that I ported back to Amidst.

Thus, if you notice strange behaviour in a current version of Amidst, please check whether the old version (Amidst v3.7) or AmidstExporter shows the same behaviour. Currently, I think that what you report is a *works as expected*.

Please let me know, if this issue can be closed.

**Comment by stefandollase** [comment]
The OP seems to have lost interest in this issue. Thus I close it.
